testsuite/src/test/java/org/jboss/da/test/server/communication/GAVComparatorRemoteTest.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package org.jboss.da.test.server.communication;
+
+import java.util.TreeSet;
+import javax.swing.event.TreeSelectionEvent;
+import org.commonjava.maven.atlas.graph.rel.SimpleDependencyRelationship;
+import org.commonjava.maven.atlas.ident.DependencyScope;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.da.model.rest.GAV;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ *
+ * @author sknot
+ */
+public class GAVComparatorRemoteTest {
+    
+
+    @Test
+    public void test() {
+        //stejne
+        GAV gav1 = new GAV("com.megginson.sax","xml-writer","0.2");
+        GAV gav2 = new GAV("com.megginson.sax","xml-writer","0.2");
+        assertTrue(gav1.compareTo(gav2) == 0);
+        
+        
+        //1 je abecedne pred, zaporne cislo
+        gav1 = new GAV("com.megginson.sax","xml-writer","0.2");
+        gav2 = new GAV("dom.megginson.sax","xml-writer","0.2");
+        assertTrue(gav1.compareTo(gav2) < 0); 
+        
+        //1 je abecedne za, kladne cislo
+        gav1 = new GAV("com.megginson.sax","xml-writer","0.2");
+        gav2 = new GAV("aom.megginson.sax","xml-writer","0.2");
+        assertTrue(gav1.compareTo(gav2) > 0);
+        
+        //random retezec
+        gav1 = new GAV("junit","junit","3.8.1");
+        gav2 = new GAV("com.thoughtworks.xstream","xstream","1.4.2");
+        assertTrue(gav1.compareTo(gav2) > 0);
+        
+        //random retezec
+        gav2 = new GAV("junit","junit","3.8.1");
+        gav1 = new GAV("com.thoughtworks.xstream","xstream","1.4.2");
+        assertFalse(gav1.compareTo(gav2) > 0);
+        
+    }
+    
+    
+
+    @Test
+    public void test2() {
+        GAV gav1 = new GAV("javax.el", "javax.el-api", "3.0.0");
+        GAV gav2 = new GAV("javax.inject", "javax.inject", "1");
+        GAV gav3 = new GAV("javax.interceptor", "javax.interceptor-api", "1.2");
+        GAV gav4 = new GAV("org.testng", "testng", "5.10");
+        GAV gav5 = new GAV("javax.ws.rs", "javax.ws.rs-api", "2.0");
+        TreeSet<GAV> set = new TreeSet<>();
+        set.add(gav1);
+        set.add(gav2);
+        set.add(gav3);
+        set.add(gav4);
+        set.add(gav5);
+        for(GAV g : set){
+            System.out.println(g);
```

Comment:
Can you rework this test so it actually test the order and doesn't print it out?